docker-compose.yml

```diff
@@ -52,6 +52,7 @@ services:
       - BYPASS_CORS=true
       - OBJECT_STORAGE_URL=nrs.objectstore.gov.bc.ca
       - OBJECT_STORAGE_ACCESS_ID=nr-fom-dev
+      - LOG_ROTATE_FILESIZE=100K
```

Comment:
Need to think about this. I'd likely prefer a daily rotation if that's an option.

Reply:
I temporarily set it to 100K for testing in Openshift. Was thinking to set it to 10M.
The reason (but not meant that I don't want) I didn't use daily was the package seem to rotate even if log is empty. There is no option to add for not rotating if empty. But that's probably small issue.